src/cl/lighthouse/lighthouse_launcher.star

```diff
@@ -348,14 +351,14 @@ def get_validator_config(
 	if len(extra_params):
 		cmd.extend([param for param in extra_params])
 
-
+	NODE_ARTIFACT_UUID = node_keystore_files.files_artifact_uuid if node_keystore_files != None else package_io.NO_ARTIFACT_UUID
```

Comment:
does no_artifact_uuid work?

I think we discsused

```py
files = {GENESIS_DATA_MOUNTPOINT_ON_CLIENTS: genesis_data.files_artifact_uuid}
if node_keystore_files:
    files[VALIDATOR_KEYS_MOUNTPOINT_ON_CLIENTS] = node_keystore_files.files_artifact_uuid
```

Reply:
lets not have capital letter variables mid functions! lets save them for global cosntants